go.mod

```diff
@@ -8,6 +8,8 @@ replace github.com/containers/image/v5 => github.com/containers/image/v5 v5.23.1
 // Pin moby/buildkit until docker/docker is upgraded
 replace github.com/moby/buildkit => github.com/moby/buildkit v0.11.6
 
+replace github.com/openshift/source-to-image => github.com/matejvasek/source-to-image v1.3.2-0.20240213191046-dee4f559d604
```

Comment:
Temporary replacement until `github.com/openshift/source-to-imag` updatest to docker v25.

Reply:
On second thought couldn't we just go with my fork? @dsimansk

Reply:
Sure, if we don't forget to remove it once s2i repo is updated. :)